starlite/app.py

```diff
@@ -6,6 +6,7 @@
 from starlette.middleware import Middleware as StarletteMiddleware
 from starlette.middleware.cors import CORSMiddleware
 from starlette.middleware.trustedhost import TrustedHostMiddleware
+from starlette.staticfiles import StaticFiles
```

Comment:
what is this import used for?

Reply:
It is used in the function body for the following check
 
 ```python
 if not isinstance(handler_fn, StaticFiles):
            raise NoRouteMatchFoundException(f"Handler with name {name} is not a static files handler")
 ```
 
The check has been introduced because I want to move actual URL generation into StaticFiles subclass to cover various cases (I am actually already doing that in my app) so since we use shared handlers index some sort of check is really needed.

Reply:
ok